### DISCUSSION TITLE
Error: netavark: nftables support presently not available

### DISCUSSION THREAD

**Discussion by composingmotion** [containers/netavark#1112] [Q&A]
I can't find exact info about this error online

Running ubuntu server 24.10 arm64

Nft service enabled and running according systemctl status 

Netavark version 1.9

Also I did the following

`sudo mkdir -p /etc/containers/containers.conf.d echo '[network] firewall_driver="nftables"' | sudo tee /etc/containers/containers.conf.d/50-netavark-nftables.conf`

Also I did a system reboot for good measure

**Comment by Luap99** [comment]
nftables support was added in v1.11

**Reply by Luap99** [reply]
1.10 actually but there have been several important bug fixes since then so I would not recommend using nftables on anything below v1.13
